### PR TITLE
fix: pages design

### PR DESCRIPTION
**Pull request by nikvoblikov**
**This PR:**
- Fixes Brand page design issues — [Problem with description](https://github.com/cilium/cilium.io/assets/105175118/2d97b457-ef93-415f-86ae-548d9e87bdc1);
- Fixes Enterprise page design issues — [Problem with mobile adaptation](https://github.com/cilium/cilium.io/assets/105175118/664f81c6-0d91-4487-b4d9-fe7e7109df44);
- Refactor all Case studies pages;

**Steps to test**

- Open the [Enterprise page](https://deploy-preview-320--cilium.netlify.app/enterprise/)
- Open the [Brand page](https://deploy-preview-320--cilium.netlify.app/brand/)
- Open the Case studies pages from [Get started page](https://deploy-preview-320--cilium.netlify.app/get-started/)
- Confirm that everything looks and works as expected

### CODE REVIEW

File: src/components/pages/brand/logos/logos.jsx [hunk 1]
```diff
@@ -43,11 +43,11 @@ const Logos = () => (
   <section className="relative bg-white pt-16 md:pt-24 lg:pt-28">
     <Container>
       <div className="flex flex-col items-start space-y-10 border-b border-gray-3 pb-10 lg:flex-row lg:justify-between lg:space-y-0">
-        <div className="flex flex-col space-y-2.5">
+        <div className="ite flex flex-col space-y-2.5">
```
Comment (vannyle) [inline src/components/pages/brand/logos/logos.jsx:46]:
```suggestion
        <div className="flex flex-col space-y-2.5">
```

File: src/components/pages/use-cases/feature-section/feature-section.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        </Heading>
+        <p className="text-lg leading-normal">{description}</p>
+      </div>
+      <div
+        className={classNames(
+          'col-span-full col-start-1 pt-6 md:col-span-8 md:col-start-3 lg:col-span-6 lg:pt-0',
+          !imageRight && 'lg:col-start-1 lg:row-start-1'
+        )}
+      >
+        {imageSrc && <img className="max-h-full lg:max-h-[350px]" src={imageSrc} alt={imageAlt} />}
```
Comment (vannyle) [inline src/components/pages/use-cases/feature-section/feature-section.jsx:46]: Can the explicit width & height be added?

Comment (vannyle) [reply src/components/pages/use-cases/feature-section/feature-section.jsx:46]: Should we use gatsby image here?

File: src/components/pages/use-cases/hero/hero.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <p className="text-lg leading-normal">{description}</p>
+      </div>
+      {(imageSrc || videoSrc) && (
+        <div
+          className={classNames(
+            'col-span-full col-start-1 pt-6 md:col-span-8 md:col-start-3 lg:col-span-6 lg:pt-14',
+            imageSrc && 'place-self-center '
+          )}
+        >
+          {imageSrc && <img className="max-h-[350px]" src={imageSrc} alt={imageAlt} />}
```
Comment (vannyle) [inline src/components/pages/use-cases/hero/hero.jsx:47]: Same question

File: src/components/pages/use-cases/join-us-cards/card/card.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    className={classNames(
+      'flex flex-col rounded-[10px] bg-white px-6 py-8 shadow-primary',
+      className
+    )}
+  >
+    <Heading className="text-center leading-tight" tag="h4" size="xs">
+      {title}
+    </Heading>
+    <p className="mt-4 mb-6 w-full" dangerouslySetInnerHTML={{ __html: description }} />
+    <a
```
Comment (vannyle) [inline src/components/pages/use-cases/join-us-cards/card/card.jsx:18]: Please you the shared component Link

File: src/components/pages/use-cases/join-us-cards/join-us-cards.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    buttonText: 'Read the Docs',
+    buttonLink: 'https://docs.cilium.io/en/stable/',
+    buttonTarget: '_blank',
+  },
+  {
+    title: 'Get Help',
+    description:
+      'Get help with Cilium through Slack, Github, training, support, and FAQs. The community can also help you tell or promote your story around Cilium.',
+    buttonText: 'Get Help',
+    buttonLink: 'https://cilium.io/get-help/',
```
Comment (vannyle) [inline src/components/pages/use-cases/join-us-cards/join-us-cards.jsx:32]:
```suggestion
    buttonLink: '/get-help/',
```

File: src/components/pages/use-cases/join-us-cards/join-us-cards.jsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    buttonTarget: '_blank',
```
Comment (vannyle) [inline src/components/pages/use-cases/join-us-cards/join-us-cards.jsx:33]:
```suggestion
```

File: src/components/pages/use-cases/use-case-card/use-case-card.jsx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+      <Heading
+        className="mb-8 max-w-full text-center leading-tight lg:mb-12 lg:max-w-[70%] lg:leading-tight xl:leading-tight"
+        tag="h2"
+        size="md"
+      >
+        {heading}
+      </Heading>
+      <ul className="flex flex-col gap-y-8 md:gap-y-10">
+        {testimonials.map(({ logo, url, CTAtext, description, title, quote, person }, index) => {
+          const Logo = logo;
```
Comment (vannyle) [inline src/components/pages/use-cases/use-case-card/use-case-card.jsx:22]: Nitpick: 
```suggestion
        {testimonials.map(({ logo: Logo, url, CTAtext, description, title, quote, person }, index) => {

```

File: src/components/pages/use-cases/use-case-card/use-case-card.jsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+          return (
+            <li
+              className="flex flex-col gap-0 rounded-lg bg-white px-6 py-8 shadow-primary sm:items-center md:flex-row md:gap-4 md:px-12"
+              key={title + index}
+            >
+              <div className="basis-2/4 items-center justify-around gap-8 md:flex">
+                <div className="flex flex-col items-center gap-6">
+                  <Logo />
+                  {url && (
+                    <a
```
Comment (vannyle) [inline src/components/pages/use-cases/use-case-card/use-case-card.jsx:32]: Same here, please use the shared Link

File: src/components/pages/use-cases/use-case-card/use-case-card.jsx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+                      href={url}
+                      target="_blank"
```
Comment (vannyle) [inline src/components/pages/use-cases/use-case-card/use-case-card.jsx:34]: The first url is linked to cilium blog, I suppose we don't need to target blank in this case

File: src/pages/use-cases/service-mesh.jsx [hunk 10]
```diff
@@ -48,21 +47,20 @@ const sectionContent4 = {
...
+    <FeatureSection {...sectionContent3} className="pb-16 md:pb-16 lg:pb-16" />
+    <FeatureSection {...sectionContent4} />
+    <JoinUsCard />
   </MainLayout>
 );
 
 export default ServiceMeshPage;
 
-export const Head = () => <title>{introContent.title}</title>;
+export const Head = () => <title>{heroContent.title}</title>;
```
Comment (vannyle) [inline src/pages/use-cases/service-mesh.jsx:66]: I believe we have the SEO component for using in Head function, please check it out https://github.com/cilium/cilium.io/blob/main/src/pages/brand.jsx#L25-L28

**Review by vannyle — APPROVED** [review]

**Review by xmulligan — APPROVED** [review]

**Pull request merged**